pkgs/servers/adminer/default.nix

```diff
@@ -0,0 +1,31 @@
+{ stdenv, libbsd, fetchurl, phpPackages, php }:
+
+stdenv.mkDerivation rec {
+  version = "4.7.6";
+  pname = "adminer";
+
+  # not using fetchFromGitHub as the git repo relies on submodules that are included in the tar file
+  src = fetchurl {
+    url = "https://github.com/vrana/adminer/releases/download/v${version}/adminer-${version}.tar.gz";
+    sha256 = "1zgvscz7jk32qga8hp2dg89h7y72v05vz4yh4lq2ahhwwkbnsxpi";
+  };
+
+  buildInputs = with phpPackages; [ php composer ];
```

Comment:
```suggestion
  nativeBuildInputs = with phpPackages; [ php composer ];
```

Reply:
I do not think this should go to `phpPackages` as it makes use of PHP but is not an extension of PHP in the same way as the other packages in `phpPackages` are. Those are mainly PECL packages, and I could not find in that list a web app like `adminer` is.

Reply:
We do have some CLI applications there but you are right that there are no web apps.